extension/js/common/inject.ts

```diff
@@ -100,4 +101,13 @@ export class Injector {
       }));
   }
 
+  private shouldInject = () => {
+    if (this.webmailName === 'gmail') {
+      if (this.S.now('gmail_logo').attr('title') === 'Chat') { // #3746
```

Comment:
This doesn't seem too robust - could we go by url starting with `https://mail.google.com/chat/` instead?

Reply:
done 👍